installation/docker.adoc

```diff
@@ -10,4 +10,6 @@
 
 toc::[]
 
-== Overview
\ No newline at end of file
+== Overview
+
+Starting in a later beta release, you will be able to install OpenShift Enterprise 3.0 Beta using Docker images. Images will be made available for OpenShift platform components such as masters, etcd, nodes, and routers.
```

Comment:
n/p. Our Jenkins preview setup will not show the Enterprise version of the docs yet; I will work on a solution to that when I am back in the US.